{"name": "test_delete_product_with_invalid_id", "status": "failed", "statusDetails": {"message": "Failed: DID NOT RAISE <class 'Exception'>", "trace": "products_table = <dbs.sqlite_products_table.SqliteProductsTable object at 0x10213d8e0>\n\n    def test_delete_product_with_invalid_id(products_table):\n        # Negative test: Attempt to delete a product with an invalid ID\n    \n        # Invalid product ID (non-existent)\n        invalid_product_id = -1\n    \n        # Get the initial count of products in the table\n        initial_product_count = len(products_table.get_all_products())\n    \n        # Attempt to delete the product with the invalid ID\n>       with pytest.raises(Exception) as exc_info:\nE       Failed: DID NOT RAISE <class 'Exception'>\n\ntest_get_products.py:74: Failed"}, "start": 1712646035627, "stop": 1712646035630, "uuid": "8e6b733d-7695-46fb-9d50-8dd834b85110", "historyId": "0e2b0779f7c39af8dad96a4fbc340434", "testCaseId": "0e2b0779f7c39af8dad96a4fbc340434", "fullName": "test.db_tests.test_get_products#test_delete_product_with_invalid_id", "labels": [{"name": "parentSuite", "value": "test.db_tests"}, {"name": "suite", "value": "test_get_products"}, {"name": "host", "value": "dariias-mbp.home"}, {"name": "thread", "value": "17395-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.db_tests.test_get_products"}]}